bundler/lib/bundler/runtime.rb

```diff
@@ -265,7 +265,7 @@ def setup_manpath
 
       return if manuals.empty?
       Bundler::SharedHelpers.set_env "MANPATH", manuals.concat(
-        ENV["MANPATH"].to_s.split(File::PATH_SEPARATOR)
+        ENV["MANPATH"] ? ENV["MANPATH"].to_s.split(File::PATH_SEPARATOR) : [""]
```

Comment:
My idea was to enhance `MANPATH`:

1. if `MANPATH` is empty, assign to `manuals` with `:` prepended
2. if `MANPATH` is not empty, just append `manuals`

Reply:
If understand this trick correctly, prepending `:` makes system man pages have precedence over gem man pages while appending it make gem man pages have precedence. So by appending, a gem overriding a potentially globally installed executable would get its man pages properly displayed.

But this is just a guess on how I would intiutively expect this to work, I need to try it and verify it.

Reply:
Ok, I added some more specs to verify that this works exactly as we want, and they confirm appending `:` to the man path when not already set does the trick.

I skipped `MANPATH` specs on Windows because I don't think Windows has `man`?

Reply:
There is no `man` on Windows, at least no by default.